recipes/robocorp-excel/README.md

```diff
@@ -0,0 +1,16 @@
+# Documentation
```

Comment:
Do you need these READMEs in the recipe? They don't seem to carry any license info or being used in the build, right?

Reply:
Hi @ocefpaf 

Thank you - I wasn't really sure how to do this correctly. 

But here is what I found while building locally. 
[pyproject.toml](https://github.com/robocorp/robo/blob/9db79af7f16bf68cdd005c3649fc6abb1f8ecc90/log/pyproject.toml#L8C1-L8C21) refers to it - and breaks while executing conda-build

On top of that all their readme in repos are linked back to the main readme at the monorepo level. Their readme in each component is a linked file.. https://github.com/robocorp/robo/tree/master/log

Here is the error I got without including readme.

'''
    File "/home/rxm7706/programs/miniconda3/conda-bld/robocorp-log_1692963088210/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p/lib/python3.11/pathlib.py", line 1044, in open
      return io.open(self, mode, buffering, encoding, errors, newline)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  **FileNotFoundError: [Errno 2] No such file or directory: '/home/rxm7706/programs/miniconda3/conda-bld/robocorp-log_1692963088210/work/README.md'**
  error: subprocess-exited-with-error

  × Preparing metadata (pyproject.toml) did not run successfully.
  │ exit code: 1
  ╰─> See above for output.

  note: This error originates from a subprocess, and is likely not a problem with pip.
'''

Reply:
Yeah. People tend to do this now and, IMO, it is a bug in the tarball b/c a necessary piece to build it is missing :-/
Leave them and, if you have the energy, open an issue upstream requesting them to put the README in the tarball so others don't have to jump through hoops to build their Software.